--- src/rviz/rviz_ogre_vendor/CMakeLists.txt
+++ src/rviz/rviz_ogre_vendor/CMakeLists.txt
@@ -7,8 +7,8 @@
 set(PACKAGE_VERSION "1.0.0")
 
 if(WIN32)
-  set(BUILDING_FREETYPE_LOCALLY ON)
-  set(BUILDING_ZLIB_LOCALLY ON)
+  set(BUILDING_FREETYPE_LOCALLY OFF)
+  set(BUILDING_ZLIB_LOCALLY OFF)
 else()
   set(BUILDING_FREETYPE_LOCALLY OFF)
   set(BUILDING_ZLIB_LOCALLY OFF)

--- src/rviz/rviz_ogre_vendor/rviz_ogre_vendor-extras.cmake.in
+++ src/rviz/rviz_ogre_vendor/rviz_ogre_vendor-extras.cmake.in
@@ -97,16 +97,7 @@
       message(FATAL_ERROR "Could not find freetype package")
     endif()
 
-    if(WIN32)
-      if(_ogre_main_static_library_debug_abs)
-        set(ZLIB_LIBRARIES "${ZLIB_HOME}/lib/zlibstaticd.lib")
-      else()
-        set(ZLIB_LIBRARIES "${ZLIB_HOME}/lib/zlibstatic.lib")
-      endif()
-    else()
-      find_package(ZLIB REQUIRED)
-    endif()
-
+    find_package(ZLIB REQUIRED)
     if(NOT ZLIB_LIBRARIES)
       message(FATAL_ERROR "Could not find zlib library")
     endif()
